From 82241148917b5beac5a6c6f47ab73b77eecc4a54 Mon Sep 17 00:00:00 2001
From: Reid Kleckner <rnk@google.com>
Date: Fri, 11 Feb 2022 10:20:10 -0800
Subject: [PATCH] [CodeView] Match any backend version in the new test

This makes the test pass for any LLVM_VERSION_MAJOR/MINOR value. Vendors
override these, and they change every six months.
---
 llvm/test/DebugInfo/COFF/compiler-version-overflow.ll | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/test/DebugInfo/COFF/compiler-version-overflow.ll b/llvm/test/DebugInfo/COFF/compiler-version-overflow.ll
index 538ba699ae6f..3374bf4519ce 100644
--- a/llvm/test/DebugInfo/COFF/compiler-version-overflow.ll
+++ b/llvm/test/DebugInfo/COFF/compiler-version-overflow.ll
@@ -3,7 +3,7 @@
 
 ; CHECK: {{.*}} | S_COMPILE3 [size = {{.*}}]
 ; CHECK-NEXT: machine = intel pentium 3, Ver = clang version 999999999999.9999999999.9999999.99999999 , language = c++
-; CHECK-NEXT: frontend = 65535.65535.65535.65535, backend = 15000.0.0.0
+; CHECK-NEXT: frontend = 65535.65535.65535.65535, backend = {{[0-9]+}}.0.0.0
 ; CHECK-NEXT: flags = none
 
 
-- 
2.35.1.473.g83b2b277ed-goog

